test/profile/myprofilegc.log.exp

```diff
@@ -1,8 +1,8 @@
 bytes allocated, allocations, type, function, file:line
            2560	             10	core.thread.Thread D main src/profilegc.d:77
             640	             10	int[] profilegc.main.bar src/profilegc.d:73
-            288	              1	immutable(char)[][int] D main src/profilegc.d:34
             240	              4	core.thread.Thread[] D main src/profilegc.d:77
+            288	              1	immutable(char)[][int] D main src/profilegc.d:34
```

Comment:
I don't understand, so, bytes allocated is how much is supposed to be allocated each time? How is something like `auto x = new int[n];` handled, since the bytes allocated on each call will vary depending on `n`?